.travis.yml

```diff
@@ -145,18 +161,18 @@ before_install:
     fi
   - python -V -V
   - python -c 'import sys; print("Python debug build:", hasattr(sys, "gettotalrefcount"))'
-  - travis_retry pip install --upgrade pip setuptools wheel
-  - travis_retry pip install cython
-  - if [ -n "$NUMPYSPEC" ]; then travis_retry pip install $NUMPYSPEC; fi
-  - travis_retry pip install --upgrade pytest pytest-xdist mpmath argparse Pillow codecov
+  # arm64 build uses conda (there aren't any arm64 wheels on PyPI, need
```

Comment:
Run a build where you build those wheels. Add pip cache into cache (it is even there already).

Do another run - pip should take those wheels from local cache.

Should work iirc.